src/libsync/syncengine.h

```diff
@@ -72,7 +73,9 @@ class OWNCLOUDSYNC_EXPORT SyncEngine : public QObject
      * -1 means infinite
      */
     void setNewBigFolderSizeLimit(qint64 limit) { _newBigFolderSizeLimit = limit; }
+    void setIgnoreHiddenFiles(bool ignore) { _csync_ctx->ignore_hidden_files = ignore; }
 
+    ExcludedFiles &excludedFiles() { return *_excludedFiles; }
```

Comment:
I think its good, saves a if(..:). @ogoffart is it more usual in other codebases? Maybe we should consider that more often.